code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -224,12 +232,24 @@ Made by Xhuis
 		if(A)
 			if(A.lighting_use_dynamic)	light_amount = T.lighting_lumcount
 			else						light_amount =  10
-		if(light_amount > 2) //Rapid death while in the light, countered by...
-			H.take_overall_damage(0,6)
+		if(light_amount > 4) //Not complete blackness - they can live in very small light levels plus starlight
```

Comment:
Use defines instead of magic numbers

at the top of the file 
define LIGHT_DAM_WEAK 4
etc etc